packages/voice/src/audio/TransformerGraph.ts

```diff
@@ -102,70 +102,28 @@ export class Node {
 
 // Create a node for each stream type
 const NODES = new Map<StreamType, Node>();
-for (const streamType of Object.values(StreamType)) {
-	NODES.set(streamType, new Node(streamType));
-}
+let hasInitializedNodes = false;
 
 /**
  * Gets a node from its stream type.
  *
  * @param type - The stream type of the target node
  */
 export function getNode(type: StreamType) {
+	if (!hasInitializedNodes) {
+		hasInitializedNodes = true;
+		for (const streamType of Object.values(StreamType)) {
+			NODES.set(streamType, new Node(streamType));
+		}
```

Comment:
If we rename `initializeNodeEdges` to `initializeNodes`, we can move this loop into the init to simplify this function.

Reply:
I'll give it a shot!

Reply:
I pushed up the changes. I need to test this some more but I think one of the issues I had with doing something similar when I first tackled this was that `initializeNodes` will be calling `getNode` internally but won't have set the map yet. What do you think?

Reply:
In that case, call `nodes.get` rather than `getNode`.

Reply:
Doh 🤦 thank you I was overthinking it. Fixed and pushed. Thank you for fixing the ESLint issue!